eventq_aws/lib/eventq_aws/aws_queue_worker.rb

```diff
@@ -236,6 +237,15 @@ def serialize_message(msg)
 
       private
 
+      def new_client_instance(options)
+        raise "[#{self.class}] - AWS Account No and Region not present." unless aws_creds_present?(options)
```

Comment:
personally, I am not a big fan of over protective software.  Plus, you raise here, but what if we make the client built upon the actual `Aws.config`, then we still raise an error here even though the client is fine.  I think it is more the responsibility of the class that needs it to raise errors.